### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by banzaman**
- This is the first time am using [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Creating an npm project with webpack. 
- Write plain HTML markup with minimum styling.
- Use ES6 modules, with import and export.

### CODE REVIEW

**Review by BregornOriginal — APPROVED** [review]
## Hi @banzaman 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 👏🏼 👏🏼 👏🏼 

## Congratulations! 

tenor-1

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Excelent pull request description
✅ Excelent work following the wireframe provided 🥇 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script><script defer src="print.bundle.js"></script></head>
+<body>
+    <h1>Leaderboard</h1>
+</body>
+</html>
```
Comment (iambenkis) [inline dist/index.html]: - [x] Kindly make sure you write the  plain HTML markup with minimum styling (just to make the layout). You should use the provided  wireframe as a guide 

Screenshot from 2023-02-01 02-10-05


# STATUS: REQUIRED CHANGES :recycle::recycle:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @banzaman, @BregornOriginal  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Webpack set :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (banzaman) [reply dist/index.html]: hey mr@ iambenkis
this is what am seeing
Screenshot (257)

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @banzaman, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**